### ISSUE TITLE
Verify FA - Turn off globally

### ISSUE THREAD

**Issue by JohnCoene** [rstudio/fontawesome#95]
My console is flooded with:

```
This Font Awesome icon ('search') does not exist:
* if providing a custom `html_dependency` these `name` checks can 
  be deactivated with `verify_fa = FALSE`
```

It's available because I imported from CDN separately (maybe not the right way about it).
Could the `verify_fa` in the `fa_i` function (used by `shiny::icon`) default to an option so we can easily turn that off globally? `fa_i(..., verify_fa = getOption("htmltools.verify_fa"))`

**Comment by rich-iannone** [comment]
Thanks for reporting this and, also, sorry about the current way that it's working.

There's no opportunity to set `verify_fa` from `shiny::icon()`, which is frustrating enough but I think your idea of having the new `verify_fa` argument in `icon()` default to a global option is a good one.

Once https://github.com/rstudio/fontawesome/pull/96 is merged and fontawesome gets a new CRAN release, I'll open a PR to make those changes in `shiny::icon()`.

**Comment by rich-iannone** [comment]
Actually, it'll be better than that. We've decided to remove `verify_fa` and icon name deprecation messages entirely. Upon merging #96 there's nothing further to do (except release to CRAN).

**Comment by daattali** [comment]
I didn't want to open a new issue to ask, but was there a regression bug in the current CRAN release (0.3.0) that causes a lot of icons to not be known? I'm getting this message even for "cog" and "sliders-h" and "file-alt", which I've been using in my code for many years and they never caused warning messages until now.

It seems the github version fixes this, but I don't see anything about it in the news so I'm trying to understand if I'm going crazy?
